node/blockchain/models/lock.py

```diff
@@ -0,0 +1,5 @@
+from djongo import models
+
+
+class TableLock(models.Model):
```

Comment:
Actually, I am not sure that we can use regular model. It would be great if we could, though. The point is that the lock must be visible outside the current transaction (also we need to figure out if Djongo actually uses MongoDB transactions). Therefore it must be set and committed in a separate database connection.